### ISSUE TITLE
Slide away the search bar when scrolling down, slide it back in when scrolling up

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1139]
This would be an intermediate step on the way to #523. It would already be a significant UI improvement in itself, and should be relatively easy to implement.
